features/capture-stream-canvas.yml

```diff
@@ -0,0 +1,14 @@
+name: captureStream() for <canvas>
+description: The `captureStream()` method for `<canvas>` elements returns a `MediaStream` which includes a `CanvasCaptureMediaStreamTrack` representing real-time video of the canvas image. You can use this to record the canvas, or send it elsewhere, such as another canvas or WebRTC connection.
+# TODO: Firefox implements `CanvasCaptureMediaStream` as an equivalent to `CanvasCaptureMediaStreamTrack`.
+#
+# Add a note when https://github.com/web-platform-dx/web-features/issues/915 is resolved.
+#
+# See also: https://bugzil.la/1231131
+spec: https://w3c.github.io/mediacapture-fromelement/#html-canvas-element-media-capture-extensions
+group: canvas
+compat_features:
+  - api.CanvasCaptureMediaStreamTrack
+  - api.CanvasCaptureMediaStreamTrack.canvas
+  - api.CanvasCaptureMediaStreamTrack.requestFrame
+  - api.HTMLCanvasElement.captureStream
```

Comment:
From our call today: let's make this key the `compute_from`—this is usable, it's just a bit of an oddity when you work with the resulting tracks, but that's probably more on the side of "supported with a caveat" not "unsupported with a circuitous workaround."